lnwallet/chainfee/estimator_test.go

```diff
@@ -131,6 +134,41 @@ func TestSparseConfFeeSource(t *testing.T) {
 	require.Error(t, err, "expected error when parsing bad JSON")
 }
 
+// TestFeeSourceCompatibility checks that when a fee source doesn't return a
+// `min_relay_feerate` field in its response, the floor feerate is used.
+//
+// NOTE: Field `min_relay_feerate` was added in v0.18.3.
+func TestFeeSourceCompatibility(t *testing.T) {
+	t.Parallel()
+
+	// Test that GenQueryURL returns the URL as is.
+	url := "test"
+	feeSource := SparseConfFeeSource{URL: url}
+
+	// Test parsing a properly formatted JSON API response.
+	//
+	// Create the resp without the `min_relay_feerate` field.
+	testFees := map[uint32]uint32{
+		1: 12345,
+	}
+	testResp := WebAPIResponse{
+		FeeByBlockTarget: testFees,
+	}
```

Comment:
this will still contain the `min_relay_feerate` field afaik. To test things when it is not there, you need to either define the anonymous struct here _or_ you need to add `omitempty` to the json tag

Reply:
Done!